webaudio/idlharness.https.html

```diff
@@ -11,27 +11,20 @@
 
 promise_test(async t => {
   const [html, dom, mediacapture, webaudio] = await Promise.all([
-    // Needed for EventTarget, HTMLMediaElement
     '/interfaces/html.idl',
-
-    // Needed for Event, EventListener
     '/interfaces/dom.idl',
-
-    // Needed for MediaStream, MediaStreamTrack
     '/interfaces/mediacapture-main.idl',
-
-    '/interfaces/webaudio.idl'
+    '/interfaces/web-audio-api.idl'
   ].map(url => fetch(url).then(response => response.text())));
 
   const idl_array = new IdlArray();
 
   // Dependencies of HTML
+  idl_array.add_untested_idls(html, { only: ['EventTarget', 'HTMLMediaElement']});
+  idl_array.add_untested_idls(dom, { only: ['Event', 'EventListener']});
+  idl_array.add_untested_idls(mediacapture, { only: ['MediaStream', 'MediaStreamTrack']});
```

Comment:
It's hard to test these changes actually work, since the test fails hard with

    promise_test: Unhandled rejection with value: object "TypeError: context.audioWorklet is undefined"

(both Chrome + Firefox)

Reply:
Ah. So the problem is the code that ends up with the `worklet_node` variable is unguarded, and if it fails the whole test does. The fix would be to guard that somehow (if or try/catch) and to not add the AudioParamMap and AudioWorkletNode instances if no instances could be created.

Reply:
Welp, you successfully deterred me from trying to clean anything up.

Reverted the change to the URL of the file only, to make this a clean PR.